Tests/BuildTests/Examples/PINRemoteImage/Pods.WORKSPACE

```diff
@@ -0,0 +1,32 @@
+new_pod_repository(
+  owner ="@ios-cx",
+  name = "PINRemoteImage",
+  url = "https://github.com/pinterest/PINRemoteImage/archive/a06b4746ebbe45c87c2b449e8a40a6b7ddf96051.zip",
+  install_script = """
```

Comment:
I think this entire install script is possible to remove as of master:
https://github.com/pinterest/PINRemoteImage/blob/master/PINRemoteImage.podspec